app/controllers/spotlight/accessibility_controller.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module Spotlight
+  ##
+  # Exhibit dashboard controller
+  class AccessibilityController < Spotlight::ApplicationController
+    before_action :authenticate_user!
+    load_and_authorize_resource :exhibit, class: Spotlight::Exhibit
+    helper_method :get_alt_info
+
+    include Spotlight::Base
+    include Spotlight::SearchHelper
+
+    def alt_text
+      authorize! :curate, @exhibit
+
+      @limit = 5
+      pages = params[:show_all] ? @exhibit.pages : @exhibit.pages.limit(@limit)
+      @pages = pages.map { |page| get_alt_info(page) }
+      @has_alt_text = @pages.sum { |page| page[:has_alt_text] }
+      @total_alt_items = @pages.sum { |page| page[:can_have_alt_text] }
+
+      attach_dashboard_breadcrumbs
+    end
+
+    def get_alt_info(page)
+      can_have_alt_text = 0
+      has_alt_text = 0
+      page.content.each do |content|
+        content.item.each_value do |item|
```

Comment:
I think you'll need account for cases where there's a widget on the page that doesn't have items? When I add a widget like the iframe widget or the header widget to a page I get an error here:

```
NoMethodError in Spotlight::AccessibilityController#alt_text
undefined method `each_value' for nil:NilClass

/spotlight/app/controllers/spotlight/accessibility_controller.rb:31:in `block in get_alt_info'
/spotlight/app/controllers/spotlight/accessibility_controller.rb:29:in `each'
/spotlight/app/controllers/spotlight/accessibility_controller.rb:29:in `get_alt_info'
/spotlight/app/controllers/spotlight/accessibility_controller.rb:19:in `block in alt_text' 
```

Reply:
Yep.